config/routes.go

```diff
@@ -66,22 +66,23 @@ func setupAPI(e *echo.Echo) {
 	d.POST("/", controllers.CreateDatacenterHandler)
 	d.PUT("/:project", controllers.UpdateDatacenterHandler)
 	d.DELETE("/:project", controllers.DeleteDatacenterHandler)
-	d.DELETE("/:project/envs/:env", controllers.DeleteServiceHandler)
-	d.POST("/:project/envs/:env/reset/", controllers.ResetServiceHandler)
-	d.DELETE("/:project/envs/:env/force/", controllers.ForceServiceDeletionHandler)
-	d.PUT("/:project/envs/:env/", controllers.UpdateServiceHandler)
-	d.POST("/:project/envs/:env/sync/", controllers.SyncServiceHandler)
-	d.GET("/:project/envs/:env", controllers.GetServiceHandler)
-	d.GET("/:project/envs/:env/builds/", controllers.GetServiceBuildsHandler)
-	d.GET("/:project/envs/:env/builds/:build", controllers.GetServiceBuildHandler)
+	d.POST("/:project/envs/", controllers.CreateEnvHandler)
+	d.DELETE("/:project/envs/:env", controllers.DeleteEnvHandler)
+	d.PUT("/:project/envs/:env/", controllers.UpdateEnvHandler)
+	d.GET("/:project/envs/:env", controllers.GetEnvHandler)
+	d.GET("/:project/envs/:env/builds/", controllers.GetEnvBuildsHandler)
+	d.GET("/:project/envs/:env/builds/:build", controllers.GetEnvBuildHandler)
+	d.POST("/:project/envs/:env/action/import/", controllers.CreateEnvHandler)
```

Comment:
[TBD] Will need some refactor on the `CreateEnvHandler` function as right now project and environment names are get from the yaml.